peekaboo/sample.py

```diff
@@ -410,9 +388,8 @@ def requested_domains(self):
     def cuckoo_report(self):
         if not self.has_attr('cuckoo_report'):
             try:
-                file_for_analysis = os.path.join(self.__wd, self.__symlink)
```

Comment:
Totally unrelated change but I'm baffled: How has this ever worked? self.__symlink is absolute already after __create_symlink() has run.

Reply:
Explained by https://docs.python.org/2/library/os.path.html#os.path.join:  If a component is an absolute path, all previous components are thrown away and joining continues from the absolute path component.

Reply:
Explains why it worked :-)